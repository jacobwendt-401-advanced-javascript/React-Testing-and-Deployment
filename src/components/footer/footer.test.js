import React from 'react';
import { shallow } from 'enzyme';

import Footer from './footer';

describe('footer.js', () => {
  it('should render a footer', () => {
    const footer = shallow(<Footer />);
    expect(footer.find('footer').exists()).toBe(true);
  });
});