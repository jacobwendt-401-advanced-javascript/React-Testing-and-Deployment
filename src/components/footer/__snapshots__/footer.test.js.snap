// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot without props 1`] = `
<footer>
  Â© 2018 Code Fellows
</footer>
`;
